test/e2e/specs/editor/blocks/columns.spec.js

```diff
@@ -43,4 +43,42 @@ test.describe( 'Columns', () => {
 		await expect( inserterOptions ).toHaveCount( 1 );
 		await expect( inserterOptions ).toHaveText( 'Column' );
 	} );
+
+	test( 'prevent the removal of locked column block from the column count change UI', async ( {
+		page,
+		editor,
+		pageUtils,
+	} ) => {
+		// Open Columns
+		await editor.insertBlock( { name: 'core/columns' } );
+		await page
+			.locator( '[aria-label="Three columns; equal split"]' )
+			.click();
+
+		// Lock last column block
+		await editor.selectBlocks(
+			page.locator( 'role=document[name="Block: Column (3 of 3)"i]' )
+		);
+		await editor.clickBlockToolbarButton( 'Options' );
+		await page.click( 'role=menuitem[name="Lock"i]' );
+		await page.locator( 'role=checkbox[name="Prevent removal"i]' ).check();
+		await page.click( 'role=button[name="Apply"i]' );
+
+		// Select columns block
+		await editor.selectBlocks(
+			page.locator( 'role=document[name="Block: Columns"i]' )
+		);
+
+		const columnsChangeInput = page.locator(
+			'role=spinbutton[name="Columns"i]'
+		);
+
+		// The min attribute should take into account locked columns
+		await expect( columnsChangeInput ).toHaveAttribute( 'min', '3' );
```

Comment:
I believe this PR introduced a flaky test in https://github.com/WordPress/gutenberg/issues/49534. @t-hamano Could you help check this :)?

Reply:
@kevin940726 
Thank you for noticing!

I was able to reproduce the same error locally as on GitHub Action. I fixed it in #49676 and it appears to be fixed locally. I will run the test several times on GitHub.